customers-service/src/main/java/com/petclinic/customers/datalayer/Photo.java

```diff
@@ -0,0 +1,42 @@
+package com.petclinic.customers.datalayer;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.util.Arrays;
+
+@Entity
+@Table(name = "photos")
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Photo {
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "type")
+    private String type;
+
+    @Column(name = "image", unique = false, nullable = false, length = 100000)
```

Comment:
Is the length amount appropriate/necessary for DB?

Reply:
useful as a failsafe in case someone inserts an absurdly large image